### PR TITLE
Add rotating carrot to dropdown and small css fixes in the remaining outstanding column

### PR DESCRIPTION
**Pull request by ysbrandB**
This makes it clearer that there is a dropdown and loses the ugly dot in front.
new:
image

### CODE REVIEW

File: resources/views/omnomcom/orders/includes/payment-details.blade.php [hunk 1]
```diff
@@ -37,40 +37,48 @@
...
-                    <div id="outstanding" class="collapse" data-parent="#outstanding-accordion">
-                        <table class="table table-borderless table-hover table-sm mt-1">
+            <div class="border-bottom-0">
+                            <span class="w-100 d-inline-flex justify-content-between">
+                                <span class=" cursor-pointer" data-bs-toggle="collapse"
+                                      data-bs-target="#collapse-outstanding">
+                                     Remaining outstanding
+                                    <i class="fas fa-info-circle" data-bs-toggle="tooltip"
+                                       data-bs-placement="right"
+                                       title="The amount caught up in activities you signed up for that still have to be processed. This will not be in the next withdrawal AND you CAN NOT pay this now, but you will have to pay this this money somewhere in the future."></i>
```
Comment (Jessseee) [inline resources/views/omnomcom/orders/includes/payment-details.blade.php:47]: This sentence is not great. Notice the double "this" in the horribly long second sentence. Maybe replace with something more concise or at least more well structured. However, I generally think this "remaining outstanding" amount is terribly confusing.

**Review by Jessseee — CHANGES_REQUESTED** [review]

File: resources/views/omnomcom/orders/includes/payment-details.blade.php (hunk 1 above)
Comment (ysbrandB) [reply resources/views/omnomcom/orders/includes/payment-details.blade.php:47]: What would you suggest to make this more clear?

Comment (ysbrandB) [reply resources/views/omnomcom/orders/includes/payment-details.blade.php:47]: Would this be better: "he amount owed for activities you signed up for that have not yet been processed by the treasurer. Therefore, you can not already pay this."?

Comment (ysbrandB) [reply resources/views/omnomcom/orders/includes/payment-details.blade.php]: We have solved this in the meeting.

**Pull request merged**